README.md

```diff
@@ -75,13 +75,13 @@ let zfsDatasetsCommand:Command = Command(bash:"zfs list -t dataset")
 let zfsProcessInterface = ProcessInterface(command:zfsDatasetsCommand, stdout:.active(.lf), stderr:.active(.unparsedRaw))
 
 //launch the process. if you are running many concurrent processes (using most of the available resources), this is where your process will be queued until there are enough resources to support the launched process.
-let outputStreams = try await zfsProcessInterface.launch()
```

Comment:
When I entered this line as-is, I get a warning saying

<img width="738" alt="CleanShot 2023-07-27 at 14 11 42@2x" src="https://github.com/tannerdsilva/SwiftSlash/assets/923876/22c4feea-f5c0-4747-945f-5fd8ae8b72a7">

> Constant 'outputStreams' inferred to have type '()', which may be unexpected

Since the `outputStreams` variable isn't being used below, I assume the intent here is just to call the `.launch()` method, and the variable isn't needed.